src/snapred/backend/recipe/algorithm/LiteDataCreationAlgo.py

```diff
@@ -40,24 +80,45 @@ def PyExec(self):
         # flag for auto deletion of non-lite data
         autoDelete = self.getProperty("AutoDeleteNonLiteWS").value
 
-        # load run number
-        run = RunConfig.parse_raw(self.getPropertyValue("RunConfig"))
-        self.chopIngredients(run)
-
         # create outputworkspace
-        outputWorkspaceName = self.getPropertyValue("OutputWorkspace")
-
-        groupingWorkspaceName = f"{self.runNumber}_lite_grouping_ws"
-
-        # load grouping map
-        self.mantidSnapper.LoadGroupingDefinition(
-            "Loading lite grouping information...",
-            GroupingFilename=str(Config["instrument.lite.map.file"]),
-            InstrumentFilename=str(Config["instrument.native.definition.file"]),
-            OutputWorkspace=groupingWorkspaceName,
-        )
-
-        self.mantidSnapper.executeQueue()
+        if self.getProperty("OutputWorkspace").isDefault:
+            # TODO use workspace name generatoe
+            outputWorkspaceName = inputWorkspaceName + "_lite"
+            self.setPropertyValue("OutputWorkspace", outputWorkspaceName)
```

Comment:
I think it may be better to just toss an exception here.  Writing the logic to discern the correct output name would probably be too complex, as the input name may have some arbitrary tags, or repeating strings, ect.  It may be easier to force the consumer to populate it, as a request for this would originate from the service layer, which would have the knowledge necessary to construct the correct output name.

Reply:
Unfortunately, mantid doesn't allow setting an _output_ workspace as mandatory.  I'm not sure why.  Yeah, I can add a validation check to ensure there is an output defined.